src/org/ods/orchestration/util/MROPipelineUtil.groovy

```diff
@@ -190,15 +193,17 @@ class MROPipelineUtil extends PipelineUtil {
                         git commit -m "${commitMessage} [ci skip]"
                         git push origin ${repo.branch}
                         """,
-                        label: "commit and push new state"
+                        label: 'commit and push new state',
+                        returnStatus: true
```

Comment:
This swallows any error. I've seen this fail in the past due to authentication, network, etc. I think it is important to keep failing the build if this does not work (same below).

Reply:
Here is what I get (sometimes ... ) and I have no idea - what that is ... (which is why I baked the returnStdout in :( )
```
+ git commit -m 'ODS: Export OpenShift configuration 
Build (assemble) #164 - Change: UNDEFINED, Project: odsst, Target Environment: odsst-dev, Version: WIP
https://..../job/odsst-cd/job/odsst-cd-repo-spock-v7-undefined-0-q/164/ [ci skip]'
# On branch master
# Untracked files:
#   (use "git add <file>..." to include in what will be committed)
#
#	../openshift-exported@tmp/
nothing added to commit but untracked files present (use "git add" to track)
```
do you see the @tmp ...

Reply:
even adding this to `.gitignore` - empty commit returns exit code 1 ... :( .. I'll add '--allow-empty' .. done :)

Reply:
`@tmp` comes from Jenkins, see https://stackoverflow.com/questions/58588794/what-are-the-tmp-folders-in-a-jenkins-workspace-and-how-to-clean-them-up and https://issues.jenkins-ci.org/browse/JENKINS-52750?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel.